pkg/sharing/member.go

```diff
@@ -165,21 +165,28 @@ func (s *Sharing) RevokeMember(inst *instance.Instance, m *Member, c *Credential
 
 	// No need to contact the revoked member if the sharing is not ready
 	if m.Status == MemberStatusReady {
-		res, err := request.Req(&request.Options{
+		opts := &request.Options{
 			Method: http.MethodDelete,
 			Scheme: u.Scheme,
 			Domain: u.Host,
 			Path:   "/sharings/" + s.SID,
 			Headers: request.Headers{
-				"Accept":       "application/vnd.api+json",
-				"Content-Type": "application/vnd.api+json",
+				"Accept":        "application/vnd.api+json",
```

Comment:
It looks like you are not using the response, so it's probably useless to put an `Accept` header